src/features/import/l10n/messageIds.ts

```diff
@@ -63,6 +63,17 @@ export default makeMessages('feat.import', {
         ),
         value: m('Value'),
       },
+      genders: {
+        gender: m('Gender'),
+        genders: {
+          f: m('female'),
+          m: m('male'),
+          null: m('unspecified'),
+          o: m('other'),
+        },
+        header: m('Gender'),
```

Comment:
Here is both a key called `header`, and `gender` on the same level translated to the same thing, "Gender". I'm thinking that both could be using the same one - where I prefer `header` since it communicates better what the transaltion is needed for? 

Also an object strucutre thing - `genders.genders` does not feel like the optimal naming, could the inner object maybe be called "genderSelectLabels" or something like that, to more communicate what they are used for?

I also think the genders need to have uppercase in the first letter to look nice in the Select 

In other parts of the code we use the word "unknown" instead of "unspecified", I think this could be good for consistency.

Reply:
Good call, I've changed them to be more descriptive I hope.